tests/dummy/app/templates/overview.hbs

```diff
@@ -113,10 +115,9 @@
       <p><a target="_BLANK" href="https://github.com/Addepar/ember-table/">Clone it from Github</a> or <a target="_BLANK" href="https://github.com/Addepar/ember-table/releases">download the ZIP repo</a></p>
       <div class="highlight">
 <pre><code>$ npm install -g grunt-cli
-$ npm install
-$ grunt
-$ node examples.js</code></pre>
-      <p>Go to your browser and navigate to <a target="_BLANK" href="http://localhost:8000/gh_pages">localhost:8000/gh_pages</a></p>
+$ npm install && bower install
```

Comment:
I prefer not putting chained instructions (using `&&` or `;`) in our docs. It's more straightforward to put each instruction on it's own line. Why is this `npm install && bower install` instead of (for instance) `npm install -g grunt-cli && npm install && bower install && ember serve`?